code/game/objects/structures/stool_bed_chair_nest_sofa/pew.dm

```diff
@@ -132,4 +132,20 @@
 
 /obj/structure/bed/chair/bench/pew/ebony
 	color    = /decl/material/solid/organic/wood/ebony::color
-	material = /decl/material/solid/organic/wood/ebony
\ No newline at end of file
+	material = /decl/material/solid/organic/wood/ebony
+
+/obj/structure/bed/chair/bench/lounge
+	name       = "lounge"
```

Comment:
to be totally honest I've never heard the word "lounge" used for a lounge chair or fancy sofa, might be an aussie thing, but it's probably fine